### ISSUE TITLE
Way to re-order items if their dom order changes

### ISSUE THREAD

**Issue by Jordan4jc** [metafizzy/packery#405]
I am using angular with an `angular-packery` module found here https://github.com/SunGard-Labs/angular-packery and using `1.4.3` of packery because of the drag order is uses over `2.0`

At the top of this plunker I have a sorting mechanism that changes the order of the items based on a property. If you turn off packery you'll see angular re-order the items properly (check the test case without packery).
However, trying to get this to work with packery I'm not finding anything that works, I have tried calling `.layout()` or `.reloadItems()` and nothing seems to be working. I assume this is intended behavior but it would be sweet if there were a way to re-order packery items, since all these items are the same width and heigh and I'm just using packery for draggabilly integration and a couple other things not really shown in this packery.

So basically I'm looking for a way to re-order the items, an animation would be really cool but not necessarily important.

**Test case without packery:** http://plnkr.co/edit/svFWGeLtM89yuoXSyPga?p=preview
**Test case with packery:** http://plnkr.co/edit/ka1VmVk0O4bs3oPdr2el?p=preview

**Comment by desandro** [comment]
Sorry, I'm not familiar with Angular so I won't be able to provide expertise there.

There is an open feature request for re-sorting the DOM with Packery dragging. You can add your 👍  to #154
